primefaces/src/main/resources/META-INF/resources/primefaces/core/core.csp.js

```diff
@@ -48,6 +52,27 @@ if (!PrimeFaces.csp) {
             }
         },
 
+        /**
+         * Checks if the given form is a Faces form.
+         * @param {HTMLInputElement} [form] The form to check.
+         * @return {boolean} true if the form is a Faces form.
+         */
+        isFacesForm: function(form) {
+            if (form.method === 'post') {
+                var children = form.children;
+                for (var i = 0; i < children.length; i++) {
+                    var child = children[i];
+                    if (child instanceof HTMLInputElement) {
+                        if (child.name.indexOf(PrimeFaces.VIEW_STATE) !== -1) {
```

Comment:
Why not use && here?

Reply:
i will provide a small refactored version; nevertheless i like loops more then lambas for such checks ;D

Reply:
No worries!